src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -173,10 +199,13 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
   return {
     selectedChannel,
+    selectedChannelKey,
     goBack,
     goToChatScreen,
+    goToMoveChat,
     goToPostDetailScreen,
     goToCommunityScreen,
+    goToContactScreen,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The returned object from `useChatUtilsHook` now includes `selectedChannelKey` and `goToContactScreen`. This change seems fine as it exposes more data and functionality to components that use this hook.


<!-- This is an auto-generated comment by OSS CodeRabbit -->